doc/source/development/contributing_codebase.rst

```diff
@@ -338,7 +338,22 @@ Writing tests
 
 All tests should go into the ``tests`` subdirectory of the specific package.
 This folder contains many current examples of tests, and we suggest looking to these for
-inspiration. Ideally, there should be one, and only one, obvious place for a test to reside.
+inspiration.
+
+As a general tip, you can use the search functionality in your integrated development
+environment (IDE) or the grep command in a terminal to find the methods you want to add a
+test for. If you are unsure of the best location to put your test, take your best guess,
+but note that reviewers may request that you move the test to a different location.
+
+To use grep, you can run the following command in a terminal:
+
+``grep -r "def function_name"``
```

Comment:
In the tests directory, we'll be looking for tests using the function rather than the definition of the function. So maybe something like `grep -r "function_name\("` instead.